entity/opencti_infrastructure.go

```diff
@@ -1,11 +1,12 @@
-// Code generated by '/tools/gocti_type_generator' for OpenCTI version 6.4.5 - DO NOT EDIT.
+// Code generated by '/tools/gocti_type_generator' for OpenCTI version 6.4.6 - DO NOT EDIT.
 
 package entity
 
 import (
 	"fmt"
 	"time"
 
+
```

Comment:
Why an addition empty line is added everywhere ? 

This change seems not legit to me, especially as it appears in almost all import blocks.

Reply:
See https://github.com/weisshorn-cyd/gocti/pull/23#discussion_r1914612024

Reply:
Ok, ty for tracking this bug into an issue